geemap/common.py

```diff
@@ -7285,20 +7285,15 @@ def ee_to_geopandas(ee_object, selectors=None, verbose=False):
         gpd.GeoDataFrame: geopandas.GeoDataFrame
     """
     from pathlib import Path
+    import geopandas as gpd
```

Comment:
Please add this line `check_package(name="geopandas", URL="https://geopandas.org")` before importing geopandas. Since geopandas is not a required dependency of geemap, the line will prompt user to install geopanas if needed. See https://github.com/giswqs/geemap/blob/master/geemap/common.py#L1010